nmostesting/Config.py

```diff
@@ -33,10 +33,10 @@
 # Number of seconds to wait after browsing for a DNS-SD advert before checking the results
 DNS_SD_BROWSE_TIMEOUT = 2
 
-# Number of seconds expected between heartbeats
+# Number of seconds expected between heartbeats for an IS-04 registry
 HEARTBEAT_INTERVAL = 5
 
-# Number of seconds to wait for the garbage collection
+# Number of seconds to wait before garbage collection for an IS-04 registry
```

Comment:
Maybe "Number of seconds an IS-04 registry is expected to wait before garbage collection"? (And just noting that similar to above, at the moment, this is only used to test a Registry, not to inform the behaviour of the mock Registries.)